src/plugin.js

```diff
@@ -55,6 +55,13 @@ export const handleEncryptedEvent = (player, event, options, sessions, eventBus)
     // return silently since it may be handled by a different system
     return Promise.resolve();
   }
+  // Legacy fairplay is the keysystem 'com.apple.fps.1_0'.
+  // If we are using this keysystem we want to use WebkitMediaKeys.
+  // This can be initialized manually with initLegacyFairplay().
+  if (options.keySystems[LEGACY_FAIRPLAY_KEY_SYSTEM]) {
```

Comment:
I see in the previous code that we checked for `keySystems` to exist. Is it possible this value will be undefined, and if so, should we check for it?

Reply:
Valid question, there's a check right above this for exactly that. Convieniently hidden when the preview of changes is collapsed. 😁 
```
if (!options || !options.keySystems) {
    // return silently since it may be handled by a different system
    return Promise.resolve();
  }
```

Reply:
ah, that makes sense :)